README.md

```diff
@@ -1,11 +1,10 @@
 # Siwe
 
-**TODO: Add description**
+Sign-In with Ethereum message validation for Elixir
```

Comment:
I would maybe provide more context links here, i.e. replace line 3 with:

Elixir library to enable [Sign-In with Ethereum](https://login.xyz) message validation; see [siwe](https://github.com/spruceid/siwe) repo for more context and interface definition for other SIWE libraries and systems.